tests/test_mixed_simulator_device.py

```diff
@@ -740,21 +740,11 @@ def circuit():
             np.random.seed(1967)
             for gates in gates_per_layers:
                 for gate in gates:
-                    params = list(np.pi * np.random.rand(gate.num_params))
-                    rnd_wires = np.random.choice(range(4), size=gate.num_wires, replace=False)
-                    gate(
-                        *params,
-                        wires=[
-                            int(w) for w in rnd_wires
-                        ]  # make sure we do not address wires as 0-d arrays
-                    )
+                    qml.apply(gate)
             return qml.expval(qml.PauliZ(0))
 
         qnode = qml.QNode(circuit, dev)
-        assert np.allclose(qnode(), 0.01160413)
-
-
-
+        assert np.allclose(qnode(), 0.0)
```

Comment:
How come the analytic value changed?

Reply:
Because the parameters were given randomly before

Reply:
Right! We did "choose" parameters after all because we seeded, correct? If we test the same behaviour, then new parameters should be good :+1:

Reply:
Yes and also because we cannot find the number of parameters that are necessary, so it is not possible to construct random circuit anymore.